root/application/src/main/java/at/fhv/itb5c/application/ApplicationFacade.java

```diff
@@ -353,4 +401,84 @@ public MatchDTO saveMatch(String sessionId, MatchDTO matchDTO, DepartmentDTO dep
 		}
 		return null;
 	}
+
+	public Boolean rsvp(String sessionId, TeamDTO team, TeamInvitationStatus answer) {
+		if (team == null || answer == null) {
+			return null;
+		}
+		if (!answer.equals(TeamInvitationStatus.Accepted) && !answer.equals(TeamInvitationStatus.Declined)) {
+			log.warn("rsvp: TeamInvitationStatus must be set to Accepted or Declined!");
+			return null;
+		}
+		try {
+			UserDTO player = getCurrentUser(sessionId);
+			Team teamEntity = ConverterTeamDTO.toEntity(team);
+			teamEntity.setMemberStatus(player.getId(), answer);
+			return PersistenceFacade.getInstance().saveOrUpdate(teamEntity) != null;
+		} catch (Exception e) {
+			log.error(e.getMessage());
+			return false;
+		}
+	}
+
+	public MessageDTO getMessage(String sessionId) {
+		QueueManager qm = new QueueManager(SessionManager.getInstance().getUserId(sessionId).toString());
+		return ConverterMessageDTO.toDTO(qm.consume());
+	}
+
+	/**
+	 * adds a player to a team from the specified tournament and creates a
+	 * notification for the player
+	 * 
+	 * @param sessionId
+	 *            session id
+	 * @param player
+	 *            player to be added
+	 * @param team
+	 *            team of the player
+	 * @param tournament
+	 *            tournament
+	 */
+	public void invitePlayer(String sessionId, UserDTO player, TeamDTO team, TournamentDTO tournament) {
+		DepartmentDTO dept = ConverterDepartmentDTO
+				.toDTO(PersistenceFacade.getInstance().getById(Department.class, tournament.getDepartmentId()));
+
+		if (hasRole(sessionId, UserRole.Admin) || isDepartmentHead(sessionId, dept)) {
```

Comment:
shouldn't the coach of the team be able to invite a player to a tournament and not the head of department